src/loc2d_ros.cpp

```diff
@@ -83,6 +83,14 @@ lama::Loc2DROS::Loc2DROS()
     pnh_.param("initial_pos_a", init_a, 0.0);
     initial_prior_ = lama::Pose2D(pos, init_a);
 
+    cur_pose_msg_.header.frame_id = global_frame_id_;
+    cur_pose_msg_.pose.pose.position.x = pos[0];
+    cur_pose_msg_.pose.pose.position.y = pos[1];
+    cur_pose_msg_.pose.pose.orientation.z = tf::createQuaternionFromYaw(init_a).getZ();
+    cur_pose_msg_.pose.pose.orientation.w = tf::createQuaternionFromYaw(init_a).getW();
+    cur_pose_msg_.header.stamp = ros::Time::now();
+    pose_pub_.publish(cur_pose_msg_);
```

Comment:
In my opinion we do not need to keep a MSG, we can create a new message each time we need to publish.
This block of code is not needed.

Reply:
I agree, I removed this since in the beginning, we are going to publish it inside `onInitialPose(const Pose2D& prior)`